torch/_higher_order_ops/effects.py

```diff
@@ -67,7 +74,10 @@ def __call__(
 with_effects = WithEffects()
 
 
-def has_aliasing(op: torch._ops.OpOverload):
+def has_aliasing(op: OpType):
+    if isinstance(op, torch._ops.HigherOrderOperator):
```

Comment:
Should we just restrict the op to be `call_torchbind`? Currently, this change seem to has larger blast radius.